Cargo.toml

```diff
@@ -21,6 +21,9 @@ ml = ["surrealdb/ml"]
 jwks = ["surrealdb/jwks"]
 performance-profiler = ["dep:pprof"]
 
+# Utility feature for tests that require Docker
+docker = []
```

Comment:
This is too visible IMHO. The comment helps but I think it might be better to use a Rust flag for this instead of a Cargo feature.